hts-precompile/TokenManagementContract.sol

```diff
@@ -0,0 +1,388 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./FeeHelper.sol";
+
+contract TokenManagementContract is FeeHelper {
+
+    string name = "tokenName";
+    string symbol = "tokenSymbol";
+    string memo = "memo";
+    uint initialTotalSupply = 1000;
+    uint32 maxSupply = 1000;
+    uint decimals = 8;
+    bool freezeDefaultStatus = false;
+
+    function createFungibleTokenPublic(
+        address treasury
+    ) public payable {
+        IHederaTokenService.TokenKey[] memory keys = new IHederaTokenService.TokenKey[](4);
+        keys[0] = getSingleKey(0, 6, 1, bytes(""));
```

Comment:
Not in this PR but we should come back to make `getSingleKey` more intuitive.
I believe there's a ticket for this #48

Reply:
Yes, I was planning to revisit this in #50 to replace those magic number together with the new `KeyHelper`.